content/chapters/data/lab/content/memory-security.md

```diff
@@ -1,16 +1,38 @@
 # Memory Security
 
-## Buffer Overflow Info Leak
+Memory security is one of the most important aspects in today's computer systems.
+The main objective of it is to prevent attackers from exploiting memory vulnerabilities that are present in a program.
```

Comment:
I'd say there are 2 objectives:
- avoid vulnerabilities altogether
- prevent attackers from exploiting the existing ones